### PR TITLE
[Beat] Reformat Phone Number in Face Payment Registration

### PR DESCRIPTION
**Pull request by lazuardyk**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?
The phone number is saved as an actual input

## What is the new behavior?
- The phone number is saved as DD{MM+YY}Phone because there is validation based on the expiration date of the visitor.
- There is a rule if the first character is 0 we will replace it with 62 because of the rules from Nodeflux's cloud.
- Added min 5 and max 15 validation

## Does this introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: backend/models/face_payment.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ type FacePaymentAccount struct {
 type NewAccountData struct {
 	SessionID string      `json:"session_id"`
 	FullName  string      `json:"full_name" validate:"required,min=2,max=255"`
-	Phone     string      `json:"phone" validate:"required,numeric"`
+	Phone     string      `json:"phone" validate:"required,numeric",min=5,max=15`
```
Comment (sonadztux) [inline backend/models/face_payment.go:24]: Eh ini min-max nya di luar tanda kutip validate bisa emang ya?

Comment (lazuardyk) [reply backend/models/face_payment.go:24]: Oiya bener juga wkaka,  efek ngoding weekend😅

Comment (lazuardyk) [reply backend/models/face_payment.go]: Fixed on https://github.com/nodefluxio/weber/pull/113/commits/b88bc66712787c46d9525f4b6fb823b4979ced43

**Review by asyafalni — APPROVED** [review]

**Review by sonadztux — APPROVED** [review]

File: backend/models/face_payment.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ type FacePaymentAccount struct {
 type NewAccountData struct {
 	SessionID string      `json:"session_id"`
 	FullName  string      `json:"full_name" validate:"required,min=2,max=255"`
-	Phone     string      `json:"phone" validate:"required,numeric"`
+	Phone     string      `json:"phone" validate:"required,numeric,min=5,max=15"`
```
Comment (sonadztux) [inline backend/models/face_payment.go:24]: Eh ini kelewat, permintaan dari PM minimal phone numbernya jadi 12 maksimal 15.

Comment (lazuardyk) [reply backend/models/face_payment.go]: Done https://github.com/nodefluxio/weber/pull/113/commits/ad0def0cc527851e83dcd2424f4528cd2b3d96c1

**Review by sonadztux — APPROVED** [review]

**Pull request merged**